fairscale/utils/testing.py

```diff
@@ -168,7 +170,7 @@ def spawn_for_all_world_sizes(test_func: Callable, world_sizes: List[int] = get_
 
 
 def worker_process(
-    rank: int, world_size: int, filename: str, filename_rpc: str, func: Callable, args: Any, error_queue: Any
+    rank: int, world_size: int, filename: str, filename_rpc: str, func: Callable, args: Any, error_queue: Any,
```

Comment:
Why the extra comma?

Reply:
autoformat, no idea

Reply:
I just removed it, not sure how that happened